name.abuchen.portfolio/src/name/abuchen/portfolio/util/HolidayType.java

```diff
@@ -30,6 +30,353 @@ public LocalDate apply(LocalDate date)
         }
     }
 
+    private static class JewishCalendarDate
+    {
+        JewishCalendarDate(int day, int month, int year)
+        {
+            this.day = day;
+            this.month = month;
+            this.year = year;
+        }
+
+        // JewishCalendarDate(JewishCalendarDate date)
+        // {
+        // this.day = date.getDay();
+        // this.month = date.getMonth();
+        // this.year = date.getYear();
+        // }
+
+        public int getDay()
+        {
+            return day;
+        }
+
+        public int getMonth()
+        {
+            return month;
+        }
+
+        public int getYear()
+        {
+            return year;
+        }
+
+        @SuppressWarnings("unused")
```

Comment:
Why suppress this warning? It is a private class. Delete the method (and we reduce noise)